includes/multi-currency/Notes/NoteMultiCurrencyAvailable.php

```diff
@@ -40,13 +41,12 @@ class NoteMultiCurrencyAvailable {
 	 * Get the note.
 	 */
 	public static function get_note() {
-		$note_class = \WC_Payment_Woo_Compat_Utils::get_note_class();
-		$note       = new $note_class();
+		$note = new Note();
```

Comment:
Do we need this variable? Can we do `return new Note()`?

Reply:
Thanks for helping me train my eyes to this 🙌 .  This is a good improvement. I will change this and take a second look for anything similar as well 👍

Reply:
From what I understand, we may need this variable, since we are modifying its properties before returning the note . example:

https://github.com/Automattic/woocommerce-payments/blob/7f0a3c6fee9ae3a917de3ef41cd38431d053b800/includes/notes/class-wc-payments-notes-loan-approved.php#L46

Apologies if I may be missing something here.

Reply:
Right, then ignore my comments.